src/tests/data/pubkey/api_sign.vec

```diff
@@ -0,0 +1,73 @@
+# [Dilithium]
```

Comment:
I know I suggested the current layout, but actually the headers would make it a bit easier to read. Maybe let's drop `Algorithm` as a key and instead use the header block to specify which algo we're testing?